platform/migration/platform-migration/pom.xml

```diff
@@ -51,6 +51,19 @@
             <artifactId>org.apache.felix.configadmin</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.apache.felix</groupId>
+            <artifactId>org.apache.felix.fileinstall</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.felix</groupId>
+            <artifactId>org.apache.felix.utils</artifactId>
```

Comment:
❓ Is it dangerous to hardcode this version here?  In other words, as we upgrade karaf/felix will we be embedding a different version than is actually used by karaf/felix?  Will we remember to update this version as we upgrade karaf/felix?

Reply:
I think I copied it from another pom file. I removed the version now.